LX.StaffScheduler.BLL/Services/Interfaces/IService.cs

```diff
@@ -0,0 +1,22 @@
+﻿using LX.StaffScheduler.BLL.DTO;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LX.StaffScheduler.BLL.Services.Interfaces
+{
+    public interface IService<T> where T : IDTO
```

Comment:
If you'd like to use IDTO anyway I would suggest to declare at least one property there which is relevant for all, public int Id {get; set;} for instance